back-end/aws-cloud9-api/src/main/java/com/serverless/QuestionHandlers/DeleteQuestionFromFormHandler.java

```diff
@@ -0,0 +1,61 @@
+package com.serverless.QuestionHandlers;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.serverless.ApiGatewayResponse;
+import com.serverless.Handler;
+import com.serverless.Response;
+import com.serverless.dal.Form;
+import com.serverless.dal.FormDBTable;
+import com.serverless.dal.Question;
+import com.serverless.dal.QuestionDBTable;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.List;
+public class DeleteQuestionFromFormHandler implements RequestHandler<Map<String, Object>, ApiGatewayResponse> {
+
+	private static final Logger LOG = LogManager.getLogger(Handler.class);
+
+	@Override
+	public ApiGatewayResponse handleRequest(Map<String, Object> input, Context context) {
+		try {
+			JsonNode body = new ObjectMapper().readTree((String) input.get("body"));
+			String questionid=(body.get("idquestion").asText());
+			QuestionDBTable question = new QuestionDBTable(); //form i table
```

Comment:
Nazwa zmiennej nie odzwierciedla co robi obiekt.Nie jest to pytanie, lecz tablica pytań.